live-examples/html-examples/inline-text-semantics/css/a.css

```diff
@@ -0,0 +1,9 @@
+a[href^="http"]::before {
+  background: url(/media/examples/external-link-alt-solid.svg) no-repeat;
+  content: '';
+  display: inline-block;
+  margin-right: 3px;
+  opacity: .6;
+  width: 0.9em;
+  height: 0.8em;
```

Comment:
Nit: Code Style: No leading zero for fraction based values.
https://stylelint.io/user-guide/rules/number-leading-zero/